specs/author_spec.rb

```diff
@@ -0,0 +1,40 @@
+require_relative '../model/author'
+require 'json'
+
+describe Author do
```

Comment:
- [ ] there are no unit tests for the `game` and `item` classes, please create unit tests for this as well.

Reply:
Well addressed this